client/wordpress-com.js

```diff
@@ -208,16 +207,11 @@ if ( config.isEnabled( 'manage/drafts' ) ) {
 }
 
 if ( config.isEnabled( 'reader' ) ) {
-	readerPaths = [
-		'/',
-		'/read'
-	];
-
 	sections.push( {
-		name: 'reader',
-		paths: readerPaths,
-		module: 'reader',
-		secondary: true,
+		name: 'reader-full-post',
+		paths: [ '/read/feeds/[^\\/]+/posts/[^\\/]+', '/read/blogs/[^\\/]+/posts/[^\\/]+' ],
```

Comment:
hrm. wasn't as easy as i hoped. 

these paths are really mostly prefixes, so the monkeying isn't that straightforward.